src/components/FieldHelp/index.js

```diff
@@ -39,6 +40,7 @@ FieldHelp.propTypes = {
   tooltip: PropTypes.string,                                          // tooltip shown when hovering the text
   children: PropTypes.any,
   container: PropTypes.any,
+  popoverWidth: PropTypes.string,
```

Comment:
Can you please add a comment here, that CSS's `width: max-content` is not supported in IE/Edge and this is a work-around for special cases?